cmd/remoteva/main.go

```diff
@@ -18,6 +19,24 @@ import (
 type Config struct {
 	RVA struct {
 		vaConfig.Common
+
+		// VerifyGRPCClientCertIfGiven, when disabled, will cause the remoteva
+		// server (which receives gRPCs from a boulder-va client) to use our
+		// default RequireAndVerifyClientCert policy. When enabled, the remoteva
+		// server will instead use the less secure VerifyClientCertIfGiven
+		// policy. It should typically be used in conjunction with the
+		// boulder-va "RVATLSClient" configuration object.
+
+		// An operator may choose to enable this if the remoteva server is
+		// logically behind an OSI layer-7 loadbalancer/reverse proxy which
+		// decrypts traffic and does not/cannot re-encrypt it's own client
+		// connection to the remoteva server.
+		//
+		// Use with caution.
+		//
+		// For more information, see: https://pkg.go.dev/crypto/tls#ClientAuthType
+		VerifyGRPCClientCertIfGiven bool
```

Comment:
This name raises the question "what's the alternative?" The answer suggested by this name is "_don't_ verify the grpc client cert", which is not the case. This should be named something that makes it clear that the default behavior is to verify.